### PR TITLE
NXLog FIM Solution v1.0.0

### PR DESCRIPTION
**Pull request by vicosurge**
Required items, please complete
   
   Change(s):
   - Initial submission for NXLog FIM Solution

   Reason for Change(s):
   - Initial submission for NXLog FIM Solution

   Version Updated:
   - No

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @v-spadarthi  , please review above PR and provide your feedback accordingly. Thanks

File: Solutions/NXLog FIM/Data/Solution_NXLogFIMTemplateSpec.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Name": "NXLog FIM",
+  "Author": "NXLog - support@nxlog.org",
+  "Description": "The NXLog [FIM](https://nxlog.co/documentation/nxlog-user-guide/im_fim.html) module allows for the scanning of files and directories, reporting detected additions, changes, renames and deletions on the designated paths through calculated checksums during successive scans. This REST API connector can efficiently export the configured FIM events to Azure Sentinel in real-time.",
+  "Data Connectors": [
+    "Data Connectors/NXLogFIM.json"
+  ],
+  "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\NXLog FIM",
+  "Version": "1.0.0",
```
Comment (v-atulyadav) [inline Solutions/NXLog FIM/Data/Solution_NXLogFIMTemplateSpec.json:9]: Hi @vicosurge , please change version as 2.0.0

File: Solutions/NXLog FIM/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		"domains" : ["IT Operations","Security - Others"],
+		"verticals": []
+	},
+	"support": {
+	  "name": "NXLog",
+	  "tier": "Partner",
+	  "link": "https://nxlog.co/user?destination=node/add/support-ticket"
+	}
+}
+
```
Comment (v-atulyadav) [inline Solutions/NXLog FIM/SolutionMetadata.json:16]: Hi @vicosurge, please remove bottom blank rows.

**Review by v-atulyadav — COMMENTED** [review]
Hi @vicosurge , please change text from `Azure Sentinel` to `Microsoft Sentinel` in data connector.

**Review by v-atulyadav — COMMENTED** [review]
Hi @vicosurge, please provide us sample data. Also add metadata in data connector file. Please refer [#link](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json) for your reference.

**Comment by v-laanjana** [comment]
Hi @vicosurge, please provide us sample data. Also add metadata in data connector file. Please refer [#link](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json) for your reference.

**Comment by vicosurge** [comment]
Hey @v-laanjana and @v-atulyadav, I no longer work at NXLog, so this case would have to be taken back to the respective members of the team over there. Thanks.

**Comment by v-sabiraj** [comment]
Hey @vicosurge, can you please tag anyone who can take it forward? thanks.

**Comment by v-atulyadav** [comment]
Hey @vicosurge, can you please tag anyone who can take it forward? thanks.

**Comment by v-laanjana** [comment]
Hey @vicosurge, can you please tag anyone who can take it forward?

**Comment by vicosurge** [comment]
@v-laanjana @v-sabiraj and @v-atulyadav, I have sent a message to the person in charge, I do not work there anymore, so I have no way of knowing which is their username.

**Comment by v-spadarthi** [comment]
Hey @vicosurge, any update whom we need to tag it ?

**Comment by vicosurge** [comment]
@v-spadarthi found the username. @techwriter-dev is John Kirch and should be the one looking into this.

**Comment by v-laanjana** [comment]
@techwriter-dev can you please update ?

**Comment by techwriter-dev** [comment]
@v-laanjana In reply to your most recent discussion:

> @techwrtier-dev can you please update?

Since the creator of this pull request ( @vicosurge ) no longer works for NXLog and is no longer a contributor to this Sentinel Solution, I have created [pull request #6264 ](https://github.com/Azure/Azure-Sentinel/pull/6264)with all the updates that have been requested in this pull request.

**Comment by v-atulyadav** [comment]
We closed this PR as new PR (pull request #6264 ) has been raised for the same by @techwriter-dev. Thanks

**Pull request closed without merging**